src/api/axios.ts

```diff
@@ -0,0 +1,8 @@
+import axios from "axios";
+
+const instance = axios.create({
+  baseURL: "https://wikied-api.vercel.app/",
+  timeout: 1000,
+});
```

Comment:
baseURL에 teamId를 추가하는 게 좋을 것 같아요!
```suggestion
const instance = axios.create({
  baseURL: "https://wikied-api.vercel.app/9-3/",
  timeout: 1000,
});
```